core/templates/dev/head/pages/exploration_player/AnswerFeedbackPairDirective.js

```diff
@@ -66,6 +66,10 @@ oppia.directive('answerFeedbackPair', [
             }
             return shortAnswerHtml;
           };
+
+          $scope.getRequestForHint = function() {
+            return $scope.data.learnerAnswer;
```

Comment:
I'm thinking about generalizing request/response as input/response. Where input refers to the learners input -- this could be either an answer or a hint request. Also, we will have to rename the answer-feedback directive to accommodate this change.